.travis.yml

```diff
@@ -76,6 +74,7 @@ script:
     - scripts/runNotebook.sh python/examples/example_*/
     - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ./scripts/run-valgrind.sh; fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ./scripts/run-doxygen.sh; fi
```

Comment:
Unrelated: I thought about splitting CI jobs. mtoc++, doxygen, ... and others which don't require AMICI compilation could be singled out and run in parallel. This could cut down travis runtime quite a bit. What do you think?

Reply:
aren't we anyways limited to 2 concurrent jobs? If not thats a valid point